src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
I'm tempted to say we should move towards not using `*Interface`, although that would introduce inconsistency